tests/main.rs

```diff
@@ -264,3 +264,24 @@ fn test_regge6j() {
         assert_eq!(vec[regge.index()], (*regge, value.clone()));
     }
 }
+
+#[test]
+fn test_wigner_12j_second() {
+    let w = Wigner12jSecond {
```

Comment:
Can the testing coverage be improved a little more? Consider either adding some more hard-coded, known cases, or add a regression test using the hash-based approach similar to `test_wigner_9j`?

Reply:
OK! I will follow your approach using hash.